### ISSUE TITLE
Fine tuning HyperParameters

### ISSUE THREAD

**Issue by CrasCris** [Deci-AI/super-gradients#1974]
### 💡 Your Question

How to find the best HyperParameters?

### Versions

I'm already do a fine tuning with a custome data set, using YOLO_NAS_S, and the results using the pretraning weights are aweason, but when i dont use them the mAP and the recall went to low, is there exist another way to find the best HyperParameters or how to improve fine tuning with a custome data set ?, (without using pretraining weights)

**Comment by BloodAxe** [comment]
You can do your own pretraining on COCO or any other dataset of your choice (PascalVOC, Objects365 etc) and then perform fine-tuning on your task. Doing so would probably improve the results on downstream task.

Finding best hyperparams is quite an open-ended problem, I suggest to look for mAP trend during training to see whether model is saturating towards the end of training or not.

Try S/M/L models with same set of hyperparams to see how big is the gap between them. If all three variants converge with a similar scores that may indicate the task is data bound. If there is a significant different between models then you can start searching for optimal LR/WD/EMA/Augmentations. Do this by turning one know at a time to see how LR changes impact the scores, then tune another parameter and so on.

**Comment by CrasCris** [comment]
Thanks for the answer :D
